buildpipeline/pipeline.json

```diff
@@ -110,7 +110,9 @@
           "Parameters": {
             "PB_DistroRid": "win7-x64",
             "PB_TargetArchitecture": "x64",
-            "PB_PortableBuild": "true"
+            "PB_PortableBuild": "true",
+            "PB_PublishRidAgnosticPackages": "true",
```

Comment:
Yes, if these don't get prefixed with `PB_`, then they become environment variables. History (CoreFx) has shown that relying on environment variables can be problematic, lead to difficult to diagnose issues, and make repro's difficult.  Our preference is to eliminate environment variables entirely, in favor of explicit declarations.  This isn't possible in VSTS, so we add a `PB_` prefix which prevents us from having variables named after something actually implemented in the code.